src/main/java/frc/robot/RobotContainer.java

```diff
@@ -88,6 +92,8 @@ public RobotContainer() {
     setTeleopDefaultCommands();
 
     smartDashSetup();
+
```

Comment:
Swap the order here. You want to register the named commands before you reference them in SmartDashSetup()